.dockerignore

```
@@ -0,0 +1,20 @@
+.dockerignore
+.git
+.github
+.gitignore
+Dockerfile
+Jenkinsfile
+Procfile
+README.md
+coverage
+docs
+features
+log
+node_modules
+package.json
+script
+spec
+test
+tmp
+vendor
+yarn.lock
```

From package.json it looks like Release uses Node modules for more than just linting, so we want `yarn install` etc. like in https://github.com/alphagov/publisher/pull/1765.

```suggestion
script
spec
test
tmp
```